public/js/showDetailBook.js

```diff
@@ -0,0 +1,31 @@
+url = window.location.pathname;
+var id = url.substring(url.lastIndexOf('/') + 1);
+$.ajax({
+    url: "/api/books/" + id,
+    type: "get",
+    success: function( data ) {
+        for(var i = 0; i <= 3; i++){
+            if(typeof data.image_books[i] === 'undefined') {
+                data.image_books[i] = {
+                    'image': '../storage/images/default-book.png'
+                };
+            }
+        }
+        let img = '<div class="product-img" style="background-image: url(' + data.image_books[0].image + ');">\
+                </div>\
+                <div class="thumb-nail">\
+                    <a href="#" class="thumb-img" style="background-image: url(' + data.image_books[1].image + ');"></a>\
```

Comment:
`data.image_books[1]` - 2- 3. Mỗi book chỉ có 3 image thôi sao. anh thấy admin cho thêm nhiều image mà?
Nếu nhiều hình thì đoạn này có thể dùng for để display